charts/consul/templates/webhook-cert-manager-clusterrole.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if or .Values.connectInject.enabled .Values.controller.enabled}}
```

Comment:
For all webhook-cert-manager helm resources, we change this to not render when the vault is enable and the following are provided:

- global.secretsBackend.vault.enabled, .global.secretsBackend.vault.enabled
- global.secretsBackend.vault.consulConnectInjectCARole
- global.secretsBackend.vault.connectInject.tlsCert.secretName
- global.secretsBackend.vault.connectInject.caCert.secretName
- global.secretsBackend.vault.consulControllerCARole
- global.secretsBackend.vault.controller.tlsCert.secretName
- global.secretsBackend.vault.controller.caCert.secretName

Since webhook cert manager manages webhook certs for controller and conect injector all of the above need to be configured together to replace the functionality.  Then webhook certs will be managed locally within controller and connect injector.